packages/fast-development-site-react/src/components/shell/pane-collapse.tsx

```diff
@@ -8,22 +9,22 @@ export interface IShellPaneCollapseProps {
 }
 
 export interface IShellPaneCollapseManagedClasses {
-    shell_pane_collapse: string;
+    shellPaneCollapse: string;
 }
 
 const style: ComponentStyles<IShellPaneCollapseManagedClasses, IDevSiteDesignSystem> = {
-    shell_pane_collapse: {
+    shellPaneCollapse: {
         background: "blue",
```

Comment:
Wondering about this value... why are we not using a design system value?

Reply:
I didn't add that value... Maybe @janechu  used it for testing something

Reply:
I can't find where this is being used at all actually.

@eljefe223 or @nalogiudice - can one of you create an Issue to investigate the usage of this and remove the entire component if necessary.